e2e-test/trace/src/integrationTest/java/com/amazon/dataprepper/integration/trace/EndToEndRawSpanTest.java

```diff
@@ -112,15 +120,17 @@ public void testPipelineEndToEnd() throws InterruptedException {
         builder.withUsername("admin");
         builder.withPassword("admin");
         final RestHighLevelClient restHighLevelClient = builder.build().createClient();
-        // Wait for otel-trace-raw-prepper by at least trace_flush_interval
-        Thread.sleep(6000);
         // Wait for data to flow through pipeline and be indexed by ES
-        await().atMost(10, TimeUnit.SECONDS).untilAsserted(
+        await().atLeast(6, TimeUnit.SECONDS).atMost(20, TimeUnit.SECONDS).untilAsserted(
```

Comment:
Why do we need to increase the time out to 20 seconds?

Reply:
This is just add some cushion for async assertion due to removal of L116: 
```
Thread.sleep(6000);
```